src/libraries/System.Reflection.Emit/ref/System.Reflection.Emit.cs

```diff
@@ -29,7 +29,9 @@ public sealed partial class AssemblyBuilder : System.Reflection.Assembly
         public System.Reflection.Emit.ModuleBuilder? GetDynamicModule(string name) { throw null; }
         [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Types might be removed")]
         public override System.Type[] GetExportedTypes() { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresAssemblyFilesAttribute("This member throws an exception for assemblies embedded in a single-file app")]
```

Comment:
NIT: wasn't it added to CodeBase too?
https://github.com/dotnet/runtime/blob/880b0d649d418d0453b3ebcf8dd44ecc19d1069e/src/libraries/System.Reflection.Emit/ref/System.Reflection.Emit.cs#L13

Reply:
I can add it there too -- this case is special-cased by the analyzer, so it's not necessary, just good style. I'll do that in a follow up PR.

Reply:
Sounds good